cs3api4lab/api/share_api_facade.py

```diff
@@ -93,7 +103,12 @@ def list_grantees_for_file(self, file_path):
         :return: list of grantees
         """
         share_list = self.share_api.list()
-        ocm_share_list = self.ocm_share_api.list()
+
+        if self._check_ocm_enabled(just_warning=True):
+            ocm_share_list = self.ocm_share_api.list()
+        else:
+            ocm_share_list = {shares: []}
```

Comment:
Shouldn't this be:
```suggestion
            ocm_share_list = {"shares": []}
```
?